### PR TITLE
revise the whole wg-evolution Chinese translations

### PR DESCRIPTION
**Pull request by eyehwan**
revise the whole wg-evolution Chinese translations upon the lastest metrics d2f9c38.
Meanwhile adding an English-Chinese comparison table.

### CODE REVIEW

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# 分支的生命周期
+
+问题:  项目如何管理其版本控制分支的生命周期?
+
+## 描述
+该指标使得版本控制分支的生命周期可见。分支生命周期包括分支创建和删除的行为，以及版本控制分支的持久化。在编写代码时，开发团队可能会创建多个分支，专注于某些特定功能。随后，这些分支可能会合并到更持久的分支，例如代码仓的主分支。一些分支在代码藏的生命周期内持续存在，而其他分支在代码合并到更持久的分支后被删除。通过了解这些模式，我们可以学习分支的创建、删除和合并如何反映特定项目的开发实践。一个代码仓的典型分支生命周期和管理方法可用于帮助识别一个项目的代码仓管理风格。
```
Comment (WUPING-888) [inline chinese/wg-evolution/code-development-activity/branch-lifecycle.md:6]: 一些分支在代码藏的生命周期内持续存在   字错误，应该是代码仓

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- 在代码仓分支很少被删除的情况下，我们如何区分分支“死亡”（即不再打算再次使用；删除）或分支“休眠”（即长时间不活动，但可能会再次使用）？
+
+## 实现
+可以在 CONTRIBUTING.md 文档中看到有关项目分支生命周期管理的所述建议。这些文档可以使用语言分析在代码仓之间进行比较，并与从实际项目实践中获得的数据进行对比，以在代码仓内部和跨代码仓中得出见解。然而，在大多数情况下，我们在这个指标中关注的数据是定量结果。
+
+**聚合器:**
+- 创建的分支数。
+- 删除的分支数。
+- 合并的分支数。
+- 放弃的分支数（有提交，但在删除之前从未合并）
```
Comment (WUPING-888) [inline chinese/wg-evolution/code-development-activity/branch-lifecycle.md:24]: 是放弃还是废弃？

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- 分支总数。
+- 分支的平均生命周期。
```
Comment (WUPING-888) [inline chinese/wg-evolution/code-development-activity/branch-lifecycle.md:26]: open branches的意思？

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- 创建分支与删除分支的比率。
+
+
+**参数：**
+- 时间段. 开始日期和完成日期。 默认：永久。
```
Comment (WUPING-888) [inline chinese/wg-evolution/code-development-activity/branch-lifecycle.md:31]: 从开始到结束日期的期间。

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- 需要统计的代码变更所处的时期段。
+
+### 筛选条件
+- 代码仓集合
+- 关于分支持久性的默认分支名称与描述性名称
+
```
Comment (WUPING-888) [inline chinese/wg-evolution/code-development-activity/branch-lifecycle.md:37]: 漏了一段：
Tools Providing the Metric (optional)
Metric must be currently deployed/available, in contrast to a tool having the "potential" to provide the metric. Provide direct link to implementation/documentation, if applicable

Augur Community Reports (https://github.com/chaoss/augur-community-reports)

Comment (eyehwan) [reply chinese/wg-evolution/code-development-activity/branch-lifecycle.md:37]: This was coming from the metric document template, so I remove it.

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md (hunk 2 above)
Comment (eyehwan) [reply chinese/wg-evolution/code-development-activity/branch-lifecycle.md:24]: I think both are fine.

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md (hunk 1 above)
Comment (eyehwan) [reply chinese/wg-evolution/code-development-activity/branch-lifecycle.md:6]: Thanks, done.

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md (hunk 4 above)
Comment (eyehwan) [reply chinese/wg-evolution/code-development-activity/branch-lifecycle.md:31]: Done.

File: chinese/wg-evolution/code-development-activity/branch-lifecycle.md (hunk 3 above)
Comment (eyehwan) [reply chinese/wg-evolution/code-development-activity/branch-lifecycle.md:26]: before it is deleted, merged or abandoned, and after it is created, we call this status of branch as open.

**Comment by eyehwan** [comment]
would be replaced by https://github.com/chaoss/translations/pull/35

**Pull request closed without merging**